From 27895714f543225b6d0d9552e885151377548800 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:10 -0400
Subject: [redhat] kabi: Adding symbol __lock_page (mm/filemap.c)

Message-id: <456c6923b7d64efadfbd607bf2774ae2aaa301f6.1410510773.git.poros@redhat.com>
Patchwork-id: 95898
O-Subject: [PATCH RHEL7.1 010/233] kabi: Adding symbol __lock_page (mm/filemap.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Get a lock on the page, assuming we need to sleep to get it.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__lock_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__lock_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__lock_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__lock_page b/redhat/kabi/kabi-module/kabi_ppc64/__lock_page
new file mode 100644
index 0000000..75bc717
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__lock_page
@@ -0,0 +1,2 @@
+#1-
+0xf9aa3ea9 __lock_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__lock_page b/redhat/kabi/kabi-module/kabi_s390x/__lock_page
new file mode 100644
index 0000000..2cfc8e4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__lock_page
@@ -0,0 +1,2 @@
+#1-
+0x149c03e2 __lock_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__lock_page b/redhat/kabi/kabi-module/kabi_x86_64/__lock_page
new file mode 100644
index 0000000..2599b1d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__lock_page
@@ -0,0 +1,2 @@
+#1-
+0xd488791e __lock_page vmlinux EXPORT_SYMBOL
-- 
1.7.1